js/components/PlaylistWindow/index.tsx

```diff
@@ -193,14 +195,8 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => {
     toggleShade: () => dispatch(togglePlaylistShadeMode()),
     scrollUpFourTracks: () => dispatch(scrollUpFourTracks()),
     scrollDownFourTracks: () => dispatch(scrollDownFourTracks()),
-    loadFilesFromReferences: (e, startIndex) =>
-      dispatch(
-        loadFilesFromReferences(
-          e.dataTransfer.files,
-          LOAD_STYLE.NONE,
-          startIndex
-        )
-      ),
+    loadMedia: (e, loadStyle, startIndex) =>
+      dispatch(loadMedia(e, loadStyle, startIndex)),
```

Comment:
Since the load style is the same, just hard code it here, rather than hard coding it when called on inside the component:

```JavaScript
    loadMedia: (e, startIndex) =>
      dispatch(loadMedia(e, LOAD_STYLE.NONE, startIndex)),
````